stl/inc/thread

```diff
@@ -311,7 +311,7 @@ public:
 
     jthread& operator=(jthread&& _Other) noexcept {
         // note: the standard specifically disallows making self-move-assignment a no-op here
-        // N4861 [thread.jthread.cons]/13
+        // N4917 33.4.4.2 [thread.jthread.cons]/13

```

Comment:
Okay this is unrelated but I dislike this behavior immensely and have filed an LWG issue.